execution_engine_testing/tests/src/test/explorer/faucet.rs

```diff
@@ -869,7 +869,7 @@ fn faucet_costs() {
     // This test will fail if execution costs vary.  The expected costs should not be updated
     // without understanding why the cost has changed.  If the costs do change, it should be
     // reflected in the "Costs by Entry Point" section of the faucet crate's README.md.
-    const EXPECTED_FAUCET_INSTALL_COST: u64 = 75_226_722_710;
+    const EXPECTED_FAUCET_INSTALL_COST: u64 = 74_833_468_520;
```

Comment:
why would transaction gossip make the faucet cost more to install? that makes no sense to me...

Reply:
The cost reduced, but that was down to the changes made in #4236, not the gossiper commit.  I'm guessing the improvement is down to the use of the new `CLValue::to_t()` avoiding some cloning.